modules/setting-up-lmeval.adoc

```diff
@@ -73,56 +72,18 @@ You can configure the following properties for LM-Eval:
 | The interval to check the job pod for an evaluation job.
 
 | `lmes-allow-online`
-| true
+| `false`
 | Whether LMEval jobs can set the online mode to `on` to access artifacts (models, datasets, tokenizers) from the internet. 
 
-| `lmes-code-execution`
-| true
+| `lmes-allow-code-execution`
+| `false`
 | Determines whether LMEval jobs can set the `trust remote code` mode to `on`.
  
 |===
 
 After updating the settings in the `ConfigMap`, restart the operator to apply the new values.
 
-
-// Notes on the 'allow online' setting and patches 
-
-// upstream - allowOnline setting is enabled; end-user can disable if they wish.
-ifdef::upstream[]
-[IMPORTANT]
---
-The `allowOnline` setting is enabled by default in {productname-long}. Using `allowOnline` gives the job permissions to automatically download artifacts from external sources. Change this setting to `false` if you do not want LM-Eval to access external sources.
---
-endif::[]
-
-// downstream - allowOnline is disabled; end-user needs include a patch in the ConfigMap in order to enable it.
-
-ifndef::upstream[]
 [IMPORTANT]
 --
-The `allowOnline` and `allowCodeExecution` settings are *disabled* by default at the operator level in {productname-long}. This disallows job permissions to automatically download artifacts from external sources. 
+The `allowOnline` and `allowCodeExecution` settings are *disabled* by default at the operator level in {productname-long}. LMEval jobs do not have internet access or permission to run any externally downloaded code unless explicitly enabled.
```

Comment:
Should "have" be changed to "allow" to mirror the [assemblies/enabling-external-resource-access-for-lmeval-jobs.adoc](https://github.com/opendatahub-io/opendatahub-documentation/pull/901/files#diff-8d340662fcfd8127657687d889523b97a8ac2fe3e10d934f0ddcad4f7fd0b7a8) abstract?